cratedb_jdbc/dialect.tdd

```diff
@@ -726,13 +726,12 @@
             <argument type='date'/>
         </function>
         <function group='aggregate' name='STDEV' return-type='real'>
-            <formula>STDDEV(%1)</formula>
+            <formula>STDDEV(%1) * sqrt(count(%1) / (count(%1) - 1.0))</formula>
```

Comment:
Do we need to check if `count > 1`?

Reply:
if count is 0 it'll return null

```sql
select
  STDDEV(a) * sqrt(count(*) / (count(*) - 1.0))
from
  unnest([]) as a
 -- null
```
```sql
select
  STDDEV(a) * sqrt(count(*) / (count(*) - 1.0))
from
  unnest([1,2]) as a
  -- 0.7...
```

Tableau doesn't test for null in the unit test so I have no way of knowing what they expect when there are no values.

Postgres does check if there is no count and returns null, but for us that behaviour is implicit
```sql
      SELECT (CASE WHEN COUNT("Calcs"."num4") = 1 THEN 0.0 WHEN COUNT("Calcs"."num4") > 0 THEN STDDEV("Calcs"."num4") ELSE NULL END)
             AS "TEMP(Test)(3542464170)(0)",
  1.1000000000000001 AS "$__alias__0"
FROM "doc"."Calcs" "Calcs"
GROUP BY 2
```

Reply:
but if the count is 1, the standard deviation is 0, and we currently return `Nan` because the whole formula resolves to `0 * infinity` which is undefined

Reply:
Thanks for explaining. I suggest that you add a variation of the above as a comment if possible